mlrun/db/httpdb.py

```diff
@@ -55,10 +55,14 @@ def bool2str(val):
     return "yes" if val else "no"
 
 
+HTTP_RETRY_AMOUNT = 3
+HTTP_RETRY_BACKOFF = 1
+HTTP_RETRY_EXCEPTIONS = (ConnectionResetError,)
```

Comment:
Please add a comment for why we are retrying on ConnectionResetError and what to take in mind when adding other exceptions.

Reply:
Even attach a link to when this error is being raised

Reply:
Can't find any "official" link for this. This is part of the TCP/IP RFC. Can link to that but it won't be very helpful...